docs/caveats.md

```diff
@@ -1,28 +0,0 @@
-# Known Limitations and Caveats
```

Comment:
this is combined with the security section; i think it makes more sense that way